connectors/opa/connector.go

```diff
@@ -83,6 +90,8 @@ func (r *ConnectorController) GetPoliciesDecisions(c *gin.Context) {
 		c.JSON(http.StatusInternalServerError, gin.H{"error": err.Error()})
 		return
 	}
+	r.log.Info().Msg(
```

Comment:
All other log messages are printed with `DEBUG` verbosity level, but this one with `INFO` , why?

Reply:
Fixed. Changed log level to Debug in opa connector and katalog-connector. Please check. Thanks.